.devcontainer/Dockerfile_celery

```diff
@@ -5,6 +5,9 @@ RUN apt-get update && apt-get install -y \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
     
+# Verify the installation of pdfinfo
+RUN pdfinfo -v
+
```

Comment:
verifying the pdfinfo is installed with poppler, because sometimes it just isn't.

Reply:
What should happen in the case that it isn't?

Reply:
It can't find pdfinfo method that's required to convert pdf into image pages, its called with poppler. So it will run into an error saying it can't find pdfinfo().